tezos/encoding/Cargo.toml

```diff
@@ -14,5 +14,8 @@ hex = "0.4"
 num-bigint = "0.3"
 num-traits = "0.2.8"
 serde = { version = "1.0", features = ["derive"] }
+nom = "6.1"
```

Comment:
@akoptelov 
please, check all Cargo.toml files and set the same version for nom,
I see somewhere `nom = "6.1.2"` and `nom = "6.1"`  (I know it is almost the same)

Reply:
Done